src/Microsoft.DotNet.Wpf/test/DRT/DrtXaml/XamlTestClasses/XamlTestClasses.csproj

```diff
@@ -4,10 +4,13 @@
     <RootNamespace>XamlTestClasses</RootNamespace>
     <Platforms>AnyCPU;x64</Platforms>
     <UseWPF>true</UseWPF>
+    <OutputType>Library</OutputType>
+    <DefineConstants>$(DefineConstants);OLDRESOURCES</DefineConstants>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   
   <PropertyGroup>
-    <SignAssembly>true</SignAssembly>
+    <SignAssembly>false</SignAssembly>
```

Comment:
Are the test assemblies getting signed? If so, what is the signing scheme being employed?